openmrs-19/src/main/resources/webapp/js/controllers.js

```diff
@@ -14,10 +14,50 @@
         $scope.newPatientIdentifierType = {};
         $scope.failureMessage = undefined;
         $scope.successMessage = undefined;
+        $scope.rollback = false;
 
-        $scope.getDefault = function() {
+        $scope.$watch('selectedConfig', function(newValue, oldValue) {
```

Comment:
All of this seem really messy. Why do we need to keep track of "originalConfig", "configOutdated" and what even is the "rollback" variable?